lib/virtual.js

```diff
@@ -48,10 +48,12 @@ function VirtualModulesPlugin(compiler) {
 
 VirtualModulesPlugin.prototype.writeModule = function(filePath, contents) {
 	var len = contents ? contents.length : 0;
-	var time = Date.now();
+	var time = new Date();
+	var timeMs = time.getTime();
 
 	var stats = new VirtualStats({
 		dev: 8675309,
+		mode: 33188,
```

Comment:
`mode` is already declared below. The duplicate declaration is making Travis fail

Reply:
Thank you, fixed. I am keeping fs.Stats property order for ease of diffing in future, if it changes.